data/json/furniture_and_terrain/terrain-manufactured.json

```
@@ -156,6 +156,54 @@
       ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_jp8_pump",
+    "name": "JP8 pump",
+    "looks_like": "t_gas_pump",
+    "description": "This pump is filled with JP8. If this gas dispenser doesn't give up the goods for free, you may have to pay at a nearby terminal.",
```

```suggestion
    "description": "This pump is filled with JP8.  If this gas dispenser doesn't give up the goods for free, you may have to pay at a nearby terminal.",
```
Is this actually true of military bases and pumps meant for military vehicles, BTW (the charging)?

---

> Is this actually true of military bases and pumps meant for military vehicles, BTW (the charging)?

It does not look like it is. I was thinking this pump might at some point be added in other places.